commands/uploader.go

```diff
@@ -270,8 +270,31 @@ func uploadPointers(c *uploadContext, unfiltered ...*lfs.WrappedPointer) {
 func (c *uploadContext) Await() {
 	c.tq.Wait()
 
+	var missing = make(map[string]string)
+	var corrupt = make(map[string]string)
+
 	for _, err := range c.tq.Errors() {
-		FullError(err)
+		if malformed, ok := err.(*tq.MalformedObjectError); ok {
+			if malformed.Missing() {
+				missing[malformed.Name] = malformed.Oid
+			} else if malformed.Corrupt() {
+				corrupt[malformed.Name] = malformed.Oid
+			}
+		} else {
+			FullError(err)
```

Comment:
What do you think about just collecting all the errors here, and not calling `FullError()`? It looks like `partitionTransfers()` can only send one other potential error:

    errors.Errorf("Git LFS: object %q has invalid size (got: %d)", t.Oid, t.Size)

Even though that's technically a server issue, I don't think it's worth halting the `push` command over. It should just be listed below.